### DISCUSSION TITLE
Integration Doc

### DISCUSSION THREAD

**Discussion by ghost** [outline/rich-markdown-editor#365] [Q&A]
Do we have to a documentation on how to create custom integration? I want to create try it out feature of swagger so that I can directly change parameters of api and test it out inside the doc instead of going to swagger.

**Comment by tommoor** [comment]
I'm sorry, I don't understand your question or what exactly you're trying to achieve. Take a look at the inbuilt extensions for examples on how to set them up:

https://github.com/outline/rich-markdown-editor/tree/develop/src/plugins
